src/graphql-proxy/recommendations/__mocks__/recommendations.ts

```diff
@@ -25,22 +26,45 @@ const fakerLocales = {
   it: 'it',
 };
 
+/**
+ *
+ * @param hasTimeToRead If true, timeToRead is set to [1, 9], otherwise it's undefined.
+ */
+const fakeRecommendation = (hasTimeToRead = true): GraphRecommendation => {
+  const recommendationWithoutTimeToRead: GraphRecommendation = {
+    __typename: 'CorpusRecommendation',
+    tileId: faker.datatype.number(),
+    corpusItem: {
+      excerpt: common.itemExcerpt(),
+      imageUrl: common.itemUrl(),
+      publisher: common.itemDomain(),
+      title: common.itemTitle(),
+      url: common.itemUrl(),
+    },
+  };
+
+  if (hasTimeToRead) {
+    return {
+      ...recommendationWithoutTimeToRead,
+      corpusItem: {
+        ...recommendationWithoutTimeToRead.corpusItem,
+        timeToRead: faker.datatype.number({ min: 1, max: 9 }),
+      },
+    };
+  } else {
+    return recommendationWithoutTimeToRead;
+  }
+};
+
```

Comment:
nit: Given that we set`hasTimeToRead` argument as `true` by default, we can flip this logic to be:

```
if(!hasTimeToRead){return recommendationWithoutTimeToRead;}
```
as an early exit. But this is fine too!